### PR TITLE
Add Retrieve Script

### PR DESCRIPTION
**Pull request by baptistebieber**
Add the functionality to retrieve data from Salesforce.

### CODE REVIEW

File: lib/plugins/gulp-jsforce-retrieve.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+var gutil = require('gulp-util');
+var through = require('through2');
+var meta = require('jsforce-metadata-tools');
+var decompress = require('decompress');
+
+/**
+ * Rework of extractZipContents from jsforce-metadata-tools to have the possibility to extract to the src folder.
+ **/
+function extractZipContents(zipFileContent, dirMapping, logger, verbose) {
+  logger.log('');
```
Comment (scolladon) [inline lib/plugins/gulp-jsforce-retrieve.js:11]: Why this logging line ?

File: lib/plugins/gulp-jsforce-retrieve.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+'use strict';
+var gutil = require('gulp-util');
+var through = require('through2');
+var meta = require('jsforce-metadata-tools');
+var decompress = require('decompress');
+
+/**
+ * Rework of extractZipContents from jsforce-metadata-tools to have the possibility to extract to the src folder.
+ **/
+function extractZipContents(zipFileContent, dirMapping, logger, verbose) {
```
Comment (scolladon) [inline lib/plugins/gulp-jsforce-retrieve.js:10]: verbose is not used.
Should be removed

**Comment by scolladon** [comment]
Retrieve task done in the master.
@baptistebieber you can delete this PR and fetch master

**Comment by baptistebieber** [comment]
Np. I do this when I've access to my computer :-)

**Pull request closed without merging**